src/os_handlers.eliom

```diff
@@ -343,6 +355,9 @@ let%shared action_link_handler _myid_o akey () =
     Eliom_registration.(appl_self_redirect Action.send) ()
   | `Custom_action_link (action_link, phantom_user) ->
     Lwt.fail (Custom_action_link (action_link, phantom_user))
+  | `Account_already_activated_unconnected (action_link) ->
+    Lwt.fail (Account_already_activated_unconnected (action_link))
```

Comment:
It should be the user...
In BS, we're going to handle it on our side.
In the template, there should be something to handle it, so we should add something to it.